etcdctl/ctlv3/command/del_command.go

```diff
@@ -16,16 +16,18 @@ package command
 
 import (
 	"fmt"
-
 	"github.com/spf13/cobra"
-	"go.etcd.io/etcd/client/v3"
+	clientv3 "go.etcd.io/etcd/client/v3"
 	"go.etcd.io/etcd/pkg/v3/cobrautl"
+	"os"
+	"time"
```

Comment:
Please try to group the import entries properly. The build-in package `os`, `time` and `fmt` should be grouped together and above other packages.